internal/graph/graph.go

```diff
@@ -117,13 +117,13 @@ func (g *ConnectedObjectGraph) findIngresses(target *openfgapb.RelationReference
 	key := tuple.ToObjectRelationString(target.GetType(), target.GetRelation())
 	if _, ok := visited[key]; ok {
 		// We've already visited the target so no need to do so again.
-		return nil, nil
+		return make([]*RelationshipIngress, 0), nil
 	}
 	visited[key] = struct{}{}
 
 	relation, err := g.typesystem.GetRelation(target.GetType(), target.GetRelation())
 	if err != nil {
-		return nil, err
+		return make([]*RelationshipIngress, 0), err
```

Comment:
Why not leave this as `nil`. The semantics were such that the return should be `nil, err` if `err` is non-nil.